### ISSUE TITLE
[BUG] Dependency specification sphinx (>=2.0<3) causes problems installing with some package managers

### ISSUE THREAD

**Issue by robna** [steven-murray/hankel#101]
**Describe the bug**
I am using [PDM](https://github.com/pdm-project/pdm) as a package manager and currently installing hankel fails due to its dependency specification `sphinx (>=2.0<3)`. I think there should be a comma between the two limiting version, see also pdm-project/pdm#2138

However, as of steven-murray@dcd5357, this should not be an issue anymore. Are you planning a new release soon, which woud include this commit?
 
**To Reproduce**
```bash
mkdir test_env
cd test_env
pdm init
pdm add hankel -v
```
```
[RequirementError]: Expected matching RIGHT_PARENTHESIS for LEFT_PARENTHESIS, after version specifier
    sphinx (>=2.0<3) ; extra == 'dev'
```

**Comment by steven-murray** [comment]
Ah! Thanks for pointing this out. Just released v1.2.2

**Comment by robna** [comment]
thanks, it works
